ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -176,4 +179,12 @@ message EnumerateComputeNodesRequest {
 message EnumerateComputeNodesResponse {
   // The list of compute nodes that have registered themselves in the current session.
   repeated ComputeNodeDescriptor compute_nodes = 1;
+}
+
+message ResolveServiceWithInformationResponse {
```

Comment:
Location wise, I would keep these messages grouped next to the other ResolveService related messages.

Reply:
Grouped with `ResolveService`.